ttnn/cpp/ttnn/operations/moreh/moreh_adam/moreh_adam_pybind.hpp

```diff
@@ -0,0 +1,90 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <pybind11/pybind11.h>
+
+#include "pybind11/cast.h"
+#include "pybind11/decorators.hpp"
+#include "ttnn/operation.hpp"
+#include "ttnn/operations/moreh/moreh_adam/moreh_adam.hpp"
+
+namespace py = pybind11;
```

Comment:
please split .hpp and .cpp
Follow ccl/all_gather for a good example.
See how .hpp only includes a lightweight pybind_fwd.hpp

Reply:
I’ve splitted the files as requested